### ISSUE TITLE
IR-RPR2-0015 unclear requirement

### ISSUE THREAD

**Issue by brfkar** [IVCTool/TS_RPR_FOM#11]
This requirment is too generic to be tested:
  
`SuT shall assume default values for optional attributes on instances of BaseEntity.PhysicalEntity and subclasses according to SISO-STD-001-2015.`

It is not possible to evaluate whether the _assumtion_ of the SuT is correct or not.

**Comment by rhzg** [comment]
I agree, optional attributes are in general not creating any observable and therefore testable reaction. This IR should be revoked.
